### PR TITLE
David/login frontend

### PR DESCRIPTION
**Pull request by sofiahowardjimenez**
## What's new in this PR? :speech_balloon:
- Finalized mongodb app set up! 
- Added blank providers for adding audio and grant queries
- Set up user auth in AuthProvider.tsx and Login.tsx
- Removed some clutter from previous templates
- Updates project README.md
### :cloud: :cloud: Documentation :cloud: :cloud:

_Describe your feature & code in detail, so that future developers know how to handle it_
- This PR sets up all necessary files to connect our app to MongoDB and our Realm database.
- To add queries, developers should go into the respective Provider in the providers folder and fill in their queries there, using realm as the database (see [this tutorial](https://www.mongodb.com/developer/products/realm/build-offline-first-react-native-mobile-app-with-expo-and-realm/#auth-provider) for examples on how to do this)
### :zap: Testing :zap:

_List all manual tests done/new unit tests you have implemented_
- In order to test, pull these changes and `run expo run:ios`. You may have to remove your node modules `rm -rf node_modules` and run `npm install`, and/or you may need to delete your ios folder `rm -rf ios` and run `expo run:ios`
- A successful simulation will build and put you on the log in screen, where you can put in a email and password, then click "sign up" -- which should lead you to the home screen
### Screenshots :calling:

_Required for visual changes - please CC Esther (leexesther) below if adding screenshots!_

## :seedling: Related PRs :seedling:

_Describe any other PRs that might affect/depend on yours_

**_Optional - any related PRs you're waiting on, or PRs that will conflict, etc_**

### :space_invader: Any notable bugs you encountered or are still having trouple with? :space_invader:
- Not sure if login/logout is working 100% of the time, some times it took me straight to the home screen upon calling `expo run:ios` -- which makes me think the user isn't getting logged out even once the program stops running? not sure, might need to look into further
### :boom: TODO:
- Additional user auth (i.e. email verification) for admins
- Styling of Login screen
- Potentially fixing the above logout bug

CC: @sofiahowardjimenez

### CODE REVIEW

**Review by varortz — APPROVED** [review]
Looks great!! Pulled the code and tested it locally, didn't have to delete any of the node modules or iOS folder.

File: RealmApp.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import Realm from "realm";
 import { APP_ID } from "@env";
-const app = new Realm.App({ id: APP_ID });
+
+const app = new Realm.App({ id: "cultural-survival-app-ugsqf" });
```
Comment (david-qing) [inline RealmApp.tsx]: Maybe we should try to set up .env if we can? We have a bit of a susceptibility here.

**Review by david-qing — APPROVED** [review]
Looking great! Tested on my computer and works perfectly. This is ready to go, as soon as we remove the susceptibility of the app_id, though I don't think it should pose a big problem.

File: RealmApp.tsx (hunk 1 above)
Comment (sofiahowardjimenez) [reply RealmApp.tsx]: That's a great thought, I'll test it right now to see if env works on mine.

**Pull request merged**